incubation-openmetrics.md

```diff
@@ -0,0 +1,46 @@
+# OpenMetrics proposal for CNCF incubation
+
+## Background
+
+OpenMetrics specifies the de-facto standard for transmitting cloud-native metrics at scale, with support for both text representation and Protocol Buffers. It’s the wire format and data model Prometheus, Cortex, and Thanos rely on and are migrating more componentes to.
+
+At the same time, OpenMetrics is pursuing an RFC under IETF OPSAWG https://datatracker.ietf.org/doc/draft-richih-opsawg-openmetrics/ . 
```

Comment:
```suggestion
At the same time, OpenMetrics is pursuing an [https://datatracker.ietf.org/doc/draft-richih-opsawg-openmetrics/](RFC under IETF OPSAWG). 
```

Reply:
That's the wrong way around, will update PR with Markdown link.